### PR TITLE
Update DSP-JS-Lib

### PR DESCRIPTION
**Pull request by kilchenmann**
- Update DSP-JS-Lib to version v1.0.0-rc.13
- Update CHANGELOG

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,45 @@
 # Changelog
 
+## v1.0.0-rc.9 (12/10/2020)
+
+## Breaking changes
+
+- #199 Integrate Ckeditor into Property View
```
Comment (tobiasschweizer) [inline CHANGELOG.md:7]: don't we link to the PRs like in dsp-js?

Comment (tobiasschweizer) [reply CHANGELOG.md:7]: I think this was https://dasch.myjetbrains.com/youtrack/issue/DSP-682, the PR's description contains additional information about the breaking change

Comment (kilchenmann) [reply CHANGELOG.md:7]: let me check...

Comment (kilchenmann) [reply CHANGELOG.md]: I had to update some titles of already closed PRs and update the "Next Release" draft. Now the changelog file should be good

**Review by tobiasschweizer — APPROVED** [review]

**Review by flavens — APPROVED** [review]

**Pull request merged**